packages/mui-material/src/ListItemText/ListItemText.d.ts

```diff
@@ -3,11 +3,44 @@ import { SxProps } from '@mui/system';
 import { InternalStandardProps as StandardProps, Theme } from '..';
 import { TypographyProps } from '../Typography';
 import { ListItemTextClasses } from './listItemTextClasses';
+import { CreateSlotsAndSlotProps, SlotProps } from '../utils/types';
+
+export interface ListItemTextSlots {
+  /**
+   * The component that renders the primary slot.
+   * @default Typography
+   */
+  primary?: React.ElementType;
+  /**
+   * The component that renders the secondary slot.
+   * @default Typography
+   */
+  secondary?: React.ElementType;
+}
+
+export type ListItemTextSlotsAndSlotProps = CreateSlotsAndSlotProps<
+  ListItemTextSlots,
+  {
+    /**
+     * Props for the primary slot
```

Comment:
```suggestion
     * Props forwarded to the primary slot (as long as `disableTypography` prop is not true).
```

Last request. I will update my PR too, `Props forwarded to the ...` sounds better than `Props for the ….`